src/masonite/api/resources/Resource.py

```diff
@@ -0,0 +1,29 @@
+from ...request import Request
+
+
+class Resource:
+    def _show(self, request: Request, user_id):
```

Comment:
I guess here we will need to specify the model as a class attribute or a method ?
```python
class UserResource(Resource):
     model = User
     # or 
     def get_model(self):
         from my.specific.path import User
         return User
```